lib/model/frames.js

```diff
@@ -189,7 +189,7 @@ Submission.Attachment = class extends Frame.define(
 class User extends Frame.define(
   table('users'), aux(Actor),
   'actorId',                            'password',
-  'mfaSecret',                          'email',        readable, writable,
+  'email',        readable, writable,
```

Comment:
I'm not sure what the spacing here signifies, so this may no longer be correct.

Reply:
I think it's meant to keep field attributes aligned across fields, sort of a column. This is the only field in the `User` frame with attributes, but I think that pattern is easier to see in other frames.

Reply:
:+1: updated to match alignment in other frames